kolibri/plugins/coach/assets/test/views/reports/ReportsExerciseLearners.spec.js

```diff
@@ -0,0 +1,163 @@
+import { shallowMount, mount } from '@vue/test-utils';
+
+import { STATUSES } from '../../../src/modules/classSummary/constants';
+import ReportsExerciseLearners from '../../../src/views/reports/ReportsExerciseLearners';
+
+const entries = [
+  {
+    id: 'd4b',
+    name: 'learner1',
+    username: 'learner1',
+    groups: ['group1', 'group2'],
+    exerciseLearnerLink: '#/2e3/reports/lessons/79b/exercises/a97/learners/d4b',
+    statusObj: {
+      learner_id: 'd4b',
+      content_id: 'a97',
+      status: STATUSES.completed,
+      last_activity: new Date('2019-05-05T07:13:22Z'),
+      time_spent: 92.5,
+    },
+  },
+  {
+    id: 'a5d',
+    name: 'learner2',
+    username: 'learner2',
+    groups: ['group2'],
+    exerciseLearnerLink: '#/2e3/reports/lessons/79b/exercises/a97/learners/a5d',
+    statusObj: {
+      learner_id: 'a5d',
+      content_id: 'a97',
+      status: STATUSES.started,
+      last_activity: new Date('2019-05-05T08:13:22Z'),
+      time_spent: 17.14,
+    },
+  },
+];
+
+const initWrapper = propsData => {
+  return mount(ReportsExerciseLearners, {
+    propsData,
+    stubs: ['router-link'],
+  });
+};
+
+const getCol = (row, colIndex) => {
+  return row.findAll('td').at(colIndex);
+};
+
+describe('ReportsExerciseLearners', () => {
+  it('smoke test', () => {
+    const wrapper = shallowMount(ReportsExerciseLearners);
+
+    expect(wrapper.isVueInstance()).toBe(true);
+  });
+
+  it('renders all entries', () => {
+    const wrapper = initWrapper({
+      entries,
+    });
+
+    const rows = wrapper.findAll({ ref: 'entry' });
```

Comment:
Could you accomplish this using a CSS selector something else?

Reply:
Answered above.